test/e2e/density.go

```diff
@@ -190,8 +190,10 @@ var _ = Describe("Density [Skipped]", func() {
 	for _, testArg := range densityTests {
 		name := fmt.Sprintf("should allow starting %d pods per node", testArg.podsPerNode)
 		if testArg.podsPerNode == 30 {
-			name = "[Performance] " + name
+			name = "[Feature:Performance] " + name
 			framework.addonResourceConstraints = density30AddonResourceVerifier()
+		} else {
+			name = "[Skipped] " + name
```

Comment:
> I'm OK with having different label for 100 pod density, and just not running it in any suite (but I guess it's pretty much equivalent to having a label Skipped/Manual).

Let's do that.  The difference is that we can scan through the `[Feature:...]`s and make sure they're running where we expect.

I'll let you & @timothysc decide about 100 enabled, etc.